crates/wit-bindgen/src/lib.rs

```diff
@@ -162,18 +162,57 @@ impl Wasmtime {
                 gen.generate_trappable_error_types(TypeOwner::Interface(*id));
                 let iface = &resolve.interfaces[*id];
 
+                let wildcard = iface.functions.values().find(|func| "*" == &func.name);
```

Comment:
While this isn't part of Wasmtime itself I think that the representation here might be better to avoid using `*` as the function's name and having it as a portion of the `functions` map. I'm worried about generators all having to have `if` clauses on the name being a bit brittle and easy to forget.

What about something like `wildcard: Option<Function>` inside of `struct Interface`? (which you may have very well already considered and rejected!)

Reply:
Absolutely agreed.  @fibonacci1729 and I were already planning to do that, so we'll make sure it happens.